osgar/drivers/usbcam.py

```diff
@@ -0,0 +1,47 @@
+"""
+  Publish jpg images captured using libuvc (pupil-labs fork).
+  Only cameras directly supporting jpg output are supported.
+"""
+
+import uvc
+from threading import Thread
+
+from osgar.bus import BusShutdownException
+
+
+class UsbCam:
+    def __init__(self, config, bus):
+        self.input_thread = Thread(target=self.run_input, daemon=True)
+        self.bus = bus
+
+        bus = config.get('bus', 1)
+        port = config.get('port', 0)
+        self.sleep = config.get('sleep', 10)
```

Comment:
one picture in 10 seconds? I would use 1 or 0.1 as default??

Reply:
yeah, that is a leftover from the time when I experimented with `framerate` instead of `sleep` :+1: